persistence/sql/persister_consent.go

```diff
@@ -47,7 +47,6 @@ func (p *Persister) revokeConsentSession(whereStmt string, whereArgs ...interfac
 			return sqlcon.HandleError(err)
 		}
 
-		var count int
```

Comment:
Actually, I don't think this should be done here, but instead checking `errors.Is(x.ErrNotFound)` here: https://github.com/ory/hydra/blob/97b1663285d323173d3581ca0ece70883bb1db3c/consent/handler.go#L115

This would also apply to your comments made in https://github.com/ory/hydra/pull/2397#issuecomment-796918032 :)

Reply:
Updated

Reply:
Could you revert the changes in this file? :) Then it is good to merge!

Reply:
done